packages/bpk-component-image/src/withLazyLoading.js

```diff
@@ -81,32 +109,40 @@ export default function withLazyLoading(Component, document) {
     // This function is taken from modernizr
     // See https://github.com/modernizr/modernizr
     // eslint-disable-next-line
-    supportsPassiveEvents() {
+    supportsPassiveEvents(): boolean {
       let supportsPassiveOption = false;
       try {
+        // $FlowFixMe
         const opts = Object.defineProperty({}, 'passive', {
           // eslint-disable-next-line getter-return
           get() {
             supportsPassiveOption = true;
+            return supportsPassiveOption;
           },
         });
         window.addEventListener('test', null, opts);
+        window.removeEventListener('test');
       } catch (e) {
         return false;
       }
       return supportsPassiveOption;
     }
 
-    isInViewPort() {
+    isInViewPort(): boolean {
+      if (documentRef.documentElement === null) {
+        return false;
+      }
       const rect = this.element.getBoundingClientRect();
 
       const viewPortHeight = Math.max(
         window.innerHeight,
-        document.documentElement.clientHeight,
+        // $FlowFixMe
```

Comment:
flow error:
```
[flow] property `clientHeight` (Property cannot be accessed on possibly null value)
```
ideas?

Reply:
Hmm, does Flow supply type information for the web browser APIs or does it just assume all unknown API calls might be null?